mappers/Baseline.rb

```diff
@@ -64,6 +64,183 @@ def initialize()
           end
         end
       end
+
+      def lookup_building_type(building_type, template, footprint_area, number_of_stories)
+        if template.include? "DEER"
+          case building_type
+          when "Education"
+            return "EPr"
+          when "Enclosed mall"
+            return "RtL"
+          when "Food sales"
+            return "RSD"
+          when "Food service"
+            return "RSD"
+          when "Inpatient health care"
+            return "Nrs"
+          when "Laboratory"
+            return "Hsp"
+          when "Lodging"
+            return "Htl"
+          when "Mixed use"
+            return "ECC"
+          when "Mobile Home"
+            return "DMo"
+          when "Multifamily (2 to 4 units)"
+            return "MFm"
+          when "Multifamily (5 or more units)"
+            return "MFm"
+          when "Nonrefrigerated warehouse"
+            return "SUn"
+          when "Nursing"
+            return "Nrs" 
+          when "Office"
+            if footprint_area
+              if footprint_area.to_f > 100000
+                return "OfL"
+              else
+                return "OfS"
+              end
+            else
+              raise "footprint_area required to map office building type"
+            end
+          when "Outpatient health care"
+            return "Nrs"
+          when "Public assembly"
+            return "Asm"
+          when "Public order and safety"    
+            return "Asm"   
+          when "Refrigerated warehouse"
+            return "WRf"
+          when "Religious worship"
+            return "Asm"
+          when "Retail other than mall"
+            return "RtS"
+          when "Service"
+            return "MLI"
+          when "Single-Family"
+            return "MFm"
+          when "Strip shopping mall"
+            return "RtL"
+          when "Vacant"
+            return "SUn"
+          else
+           raise "building type #{building_type} cannot be mapped to a DEER building type"
+          end
+
+        else
```

Comment:
@kflemin I guess this mapping is used even when no template is specified?

Reply:
if no template is specified (or it is nil), then it assumes the ASHRAE types like it used to.